src/js/FirmwareCache.js

```diff
@@ -168,13 +169,39 @@ let FirmwareCache = (function () {
         });
     }
 
+    /**
+     * Remove all cached data
+     */
+    function destroy() {
```

Comment:
If this is a method that is going to be called from the console, maybe is better to give it a more intuitive method. A `destroy()` is maybe a little catastrophic and something like `cleanFirmwareCache()` or similar is more intuitive.